packages/sdk-components-react-radix/src/tabs.ws.ts

```diff
@@ -79,19 +79,7 @@ export const metaTabs: WsComponentMeta = {
     {
       type: "instance",
       component: "Tabs",
-      variables: {
-        tabsValue: { initialValue: "0" },
-      },
-      props: [
-        { type: "expression", name: "value", code: "tabsValue" },
-        {
-          name: "onValueChange",
-          type: "action",
-          value: [
-            { type: "execute", args: ["value"], code: `tabsValue = value` },
-          ],
-        },
-      ],
+      props: [{ type: "string", name: "defaultValue", value: "0" }],
```

Comment:
Perhaps we should stop ignoring defaultValue/defaultOpen in generated props. See package.json in radix sdk.

Reply:
We are showing them in template
<img width="315" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/5ba108e5-a46d-4ccb-a8d4-d2fb5cb1db71">